lms/djangoapps/courseware/transformers.py

```diff
@@ -0,0 +1,93 @@
+"""
```

Comment:
I didn't think adding `grades_` would add much value, as the class name already contains that information.  Also, if anything else were added to the file, it would probably not be grades related; this provides more flexibility to delay refactoring if appropriate when another transformer is added.